src/main/resources/view/MainWindow.fxml

```diff
@@ -51,39 +51,31 @@
                                       </children>
                                   </VBox>
                               </Tab>
-                              <Tab fx:id="tabExpired" closable="false" text="Expired" disable="true">
-                                  <VBox fx:id="expiredTaskList" maxHeight="Infinity" minWidth="200.0" prefWidth="200" VBox.vgrow="ALWAYS">
+                              <Tab fx:id="tabExpired" closable="false" disable="true" text="Expired">
+                                  <VBox fx:id="expiredTaskListPanel" layoutX="10.0" layoutY="157.0"
+                                        maxHeight="Infinity" minWidth="200.0" prefWidth="200" style="-fx-background-radius: 20;">
                                       <padding>
                                           <Insets bottom="5" left="5" right="5" top="5" />
                                       </padding>
-                                      <StackPane fx:id="expiredTaskListPanelPlaceholder" VBox.vgrow="ALWAYS">
-                                          <VBox.margin>
-                                              <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
-                                          </VBox.margin>
+                                      <StackPane fx:id="expiredTaskListPanelPlaceholder" style="-fx-background-radius: 20;" VBox.vgrow="ALWAYS">
                                       </StackPane>
                                   </VBox>
                               </Tab>
-                              <Tab fx:id="tabCompleted" closable="false" text="Completed" disable="true">
+                              <Tab fx:id="tabCompleted" closable="false" disable="true" text="Completed">
                                   <VBox fx:id="completedTaskListPanel" maxHeight="Infinity" minWidth="200.0" prefWidth="200" VBox.vgrow="ALWAYS">
                                       <padding>
                                           <Insets bottom="5" left="5" right="5" top="5" />
                                       </padding>
                                       <StackPane fx:id="completedTaskListPanelPlaceholder" VBox.vgrow="ALWAYS">
-                                          <VBox.margin>
-                                              <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
-                                          </VBox.margin>
                                       </StackPane>
                                   </VBox>
                               </Tab>
-                              <Tab fx:id="tabUncompleted" closable="false" text="Uncompleted" disable="true">
+                              <Tab fx:id="tabUncompleted" closable="false" disable="true" text="Uncompleted">
```

Comment:
What's the difference haha?

Reply:
I think scene builder automatically chnaged lol